packages/@react-spectrum/tree/src/TreeView.tsx

```diff
@@ -66,6 +66,10 @@ const TreeRendererContext = createContext<TreeRendererContextValue>({});
 // keyboard focus ring. Perhaps find a different way of rendering the outlines since the top of the item doesn't
 // scroll into view due to how the ring is offset. Alternatively, have the tree render the top/bottom outline like it does in Listview
 const tree = style<Pick<TreeRenderProps, 'isEmpty'>>({
+  ...focusRing(),
+  outlineOffset: '[-2px]', // make certain we are visible inside overflow hidden containers
```

Comment:
```suggestion
  outlineOffset: -2, // make certain we are visible inside overflow hidden containers
```

arbitrary value syntax shouldn't be needed?

Reply:
this is v3, we didn't update it alongside the change in s2 to allow negative values
I could do that too, just didn't want to end up updating everything about the v3 macros theme in this PR